velox/exec/fuzzer/PrestoQueryRunner.h

```diff
@@ -100,6 +109,14 @@ class PrestoQueryRunner : public velox::exec::test::ReferenceQueryRunner {
 
   bool supportsVeloxVectorResults() const override;
 
+  // Traverses all nodes in the plan and returns all tables and their names.
+  std::unordered_map<std::string, std::vector<velox::RowVectorPtr>>
+  getAllTablesAndNames(const core::PlanNodePtr& plan);
+
+  std::vector<velox::RowVectorPtr> executeVector(
```

Comment:
nit: Add a comment , also maybe rename as executeAndReturnVector , because it reads like it should take a vector as argument..

Reply:
Done.